test/Tizen.NUI.PerformanceTest/tizen-manifest.xml

```diff
@@ -1,16 +1,16 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns="http://tizen.org/ns/packages" api-version="6" package="org.tizen.example.Tizen.NUI.PerfNew" version="1.0.0">
+<manifest xmlns="http://tizen.org/ns/packages" api-version="6" package="org.tizen.example.Tizen.NUI.PerformanceTest" version="1.0.0">
   <profile name="common" />
-  <ui-application appid="org.tizen.example.Tizen.NUI.PerfNew"
-					exec="PerfNew.dll"
+  <ui-application appid="org.tizen.example.Tizen.NUI.PerformanceTest"
+					exec="PerformanceTest.dll"
 					type="dotnet"
```

Comment:
from next time, please use "dotnet-nui" to activate "preloading"
thank you.

Reply:
Fix done. thank you.